components/src/core/components/Input/Color/ColorPicker.vue

```diff
@@ -0,0 +1,161 @@
+<!--
+/*
+ * This file is part of OrangeHRM Inc
+ *
+ * Copyright (C) 2020 onwards OrangeHRM Inc
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see  http://www.gnu.org/licenses
+ */
+-->
+
+<template>
+  <div role="alert" class="oxd-color-picker">
+    <canvas
+      ref="colorPalette"
+      width="230"
+      height="230"
+      class="oxd-color-picker-palette"
+      @mousemove.prevent="pickColor"
+      @touchmove.prevent="pickColor"
+      @mousedown.prevent="captureOn"
+      @touchstart.prevent="captureOn"
+    ></canvas>
+    <input
+      v-model.lazy="hue"
+      class="oxd-color-picker-range"
+      type="range"
+      max="359"
+    />
+    <oxd-label
+      :label="t('general.hex', 'HEX')"
```

Comment:
HEX is a code and I don't think it needs to be localized

Reply:
Possibly yes. but it will fallback if not defined.